### ISSUE TITLE
update from 2.5.0 to 2.6.0 break pw-fdo

### ISSUE THREAD

**Issue by lumag** [getpatchwork/pwclient#23]
Updating pwclient from 2.5.0 to 2.6.0 broke updating pathches in pw-fdo setup:

```
$  ~/.local/bin/pwclient update -s Accep 524726
Patch not updated
```
Bisecting pointed out to 0949e9e5cf788f6092ebfc3bef6b9cdc014efd8d as the first bad commit.

**Comment by stephenfin** [comment]
If it hasn't raised an exception then it's doing the correct thing. You can ignore that error message: I'll remove it shortly.

**Comment by stephenfin** [comment]
Fixed in [2.6.2](https://github.com/getpatchwork/pwclient/releases/tag/2.6.2).
